test/adapters/parsers/csv.spec.js

```diff
@@ -119,4 +120,56 @@ describe('parsers/csv', function() {
         });
     });
 
+    var pointFileTSV = path.resolve(__dirname, 'input/tsv/point.tsv');
+    var pointsFileTSV = path.resolve(__dirname, 'input/tsv/points.tsv');
+    var invalidFileTSV = path.resolve(__dirname, 'input/tsv/invalid.tsv');
+
+    _.each({
```

Comment:
`_.each` of one entry seems like overkill, no?

For completeness we should test that it works fine when you explicitly give a `,` as the separator. Then you could parameterize the extension.